src/index.html

```diff
@@ -123,9 +123,15 @@ <h2>&rarr; Twitter</h2>
 
       <h1>Code of Conduct</h1>
 
-      <p><strong>All</strong> of our events follow a code of conduct that you can read at: <a href="https://github.com/jsoxford/summerofhacks.github.io/blob/master/CODEOFCONDUCT.md">github.com/jsoxford/summerofhacks.github.io/blob/master/CODEOFCONDUCT.md</a></p>
-
-      <p>It's our absolute priority that attendees are safe and are feel respected. If you have any issues, concerns or questions then don't hesitate to contact us.</p>
+      <p><strong>All</strong> of our events follow a code of conduct that you can read at
+        <a href="https://github.com/summer-of-hacks/soh19website/blob/master/CODEOFCONDUCT.md">
+          https://github.com/summer-of-hacks/soh19website/blob/master/CODEOFCONDUCT.md</a></p>
+
+      <p>It's our absolute priority that attendees are safe and feel respected.
+        If you have any issues, concerns, or questions then contact us:
+        organisers will share contact details at each event so that all attendees
+        will have a number of ways to raise issues or concerns both on the day
+        or after the event.</p>
```

Comment:
Thoughts on this one?

I made this change after reading somewhere—sorry, cannot remember where—that some folk prefer a digital, non-confrontational way to communicate issues relating to the COC. Not everyone is cool with physically speaking to someone about a (potentially) troubling issue.

Reply:
yep, very fine with that.